### DISCUSSION TITLE
Background voxels and 'unlabeled' interior voxels

### DISCUSSION THREAD

**Discussion by paul-reiners** [MIC-DKFZ/nnUNet#650] [Q&A] [answered]
For labelled images (those in the _labelsTr_ folder), do we need to distinguish between background voxels with a value of 0, and unlabeled or unknown interior voxels?  In particular, I'm working with brain MRI images.  Should **only** parts of the image external to the brain be labeled 0 or can there be voxels interior to the brain that are labeled 0 (perhaps because the labeler used 0 for parts of the brain they couldn't identify)?

**Comment by FabianIsensee** [comment, marked as answer]
background should be 0. There is no support for unlabeled voxels
Best,
Fabian
